content/docs/authorization/Migration guide from v1 to v2/_index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Authorization - v2 Migration guide
+linktitle: Migration guide from v1 to v2
+weight: 1
+description: >
+  CSM for Authorization v1 to v2 Migration guide
+---
+
+CSM for Authorization v2 has major architectural changes due to which user will not be able to upgrade CSM for Authorization v1 to CSM for Authorization v2. This page provide a referent guide to user **How to migrate** from version 1 to version 2 release.
```

Comment:
Noticed some minor issues - Can you update this to say "CSM for Authorization v2 has significant architectural changes that prevent a user from upgrading CSM Authorization v1 to CSM Authorization v2. This page is a reference guide for migrating an existing v1 deployment to v2."